mappings/servers/kohilol/metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+    "id": "kohilol",
+    "name": "KohiLOL Network",
+    "description": "Remake of kohi.",
+    "addresses": [
+        "kohi.lol",
+        "na.kohi.lol",
+        "as.kohi.lol",
+        "asia.kohi.lol"
+    ],
+    "primaryAddress": "kohi.lol",
+    "minecraftVersions": [
+        "1.7.*",
+        "1.8.*"
+    ],
+    "primaryMinecraftVersion": "1.7.10",
+    "primaryColor": "#05fa0d",
+    "secondaryColor": "#faf205",
+    "primaryLanguage": "en",
+    "languages": [
+        "en",
+        "es-EN"
+    ],
+    "primaryRegion": "NA",
+    "regions": [
+        "NA",
+        "ASIA"
```

Comment:
`ASIA` is not part of our scheme. Refer to https://github.com/CheatBreakerNet/Client-API/blob/e3fc3606ad3d3c3dcf9404b1c53bb17ab643f2e9/mappings/metadata.schema.json#L152 - `AS` is more appropriate

Reply:
changed.